### DISCUSSION TITLE
What are some creative integrations using the API?

### DISCUSSION THREAD

**Discussion by jh20001** [YOURLS/YOURLS#2920] [General]
I have been trying to rack my mind to come up with some creative integrations. 3rd party services/websites (ie, IFTTT) that can make use of YOURLS. I thought of Wordpress, but after looking into things, the one plugin that seemed to support it hasn't been updated in a few years or more, and there is only one screenshot that doesn't show much. 

Just wondering what ideas or solutions others have come up with or ran into recently :)

**Comment by ozh** [comment]
AFAIK there are several WP plugins using YOURLS. There are also numerous known integrations with other languages or platforms, see https://github.com/YOURLS/awesome-yourls#integrations-

**Reply by jh20001** [reply]
Yeah, I have seen a lot of those. I was discussing it with some individuals within the WP dev community and they had said almost all of the YOURLS related plugins haven't been updated in 2-5+ years and are no longer supported by their authors. It would be nice to ThirstyAffiliates picked up support for YOURLS integration. Now that would make for a powerful partnership there. 

Thanks for the link though. I guess I never made it that far down that page. I re-did a search and found one (NextScripts) that offers support for YOURLS. That's good to see at least one I recognize. Some of the plugins offer lightly mentioned support that isn't that obvious. 

I wonder about other integrations others have run into though. Ex, I see that TweetDeck offers support for it (sadly CrowdFire does not). 

I keep souring Google for references, lists, etc (ex, "top YOURLS integrations"). None of it returns a single thing. Only alternatives to YOURLS, etc, which I am not looking for. Google is usually really good, but it still sometimes fails you.  *sigh*
